example/presto_standalone/main.tf

```diff
@@ -1,58 +1,44 @@
 locals {
-  nomad_namespace   = "default"
   nomad_datacenters = ["dc1"]
-
-  presto = {
-    use_vault_secret_provider = true
-    vault_kv_policy_name      = "kv-secret"
-    vault_kv_path             = "secret/data/dev/presto"
-    vault_kv_secret_key_name  = "cluster_shared_secret"
-    service_name              = "presto"
-  }
-
-  hivemetastore = {
-    service_name = module.hive.service_name
-    port         = module.hive.port
-  }
-
-  minio = {
-    service_name = module.minio.minio_service_name
-    port         = module.minio.minio_port
-    access_key   = module.minio.minio_access_key
-    secret_key   = module.minio.minio_secret_key
-  }
+  nomad_namespace   = "default"
```

Comment:
I can see the convenience, but shouldn't all locals be removed?

Reply:
Well, originally I understood the reasoning to be a wish for better readability. In that case I'd argue it is more readable the way it is now, because then it is clear all the modules are set up in the same namespace and datacent

Reply:
It's already so much better to read so I don't mind having those as locals.

Reply:
@zhenik @dangernil any input?

Reply:
I gave up

Reply:
Gave up on what?

Reply:
I agree with @claesgill I see the convenience. So, its ok.

Reply:
Sweet. I'm still confused though, what did you give up on? Arguing with me? 😅